lib/generate.ex

```diff
@@ -172,11 +172,12 @@ defmodule PuppeteerPdf.Generate do
             {:error, message}
 
           _ ->
+            {command, options} = CommandHelper.get_command(exec_path, params)
```

Comment:
I think we can put `System.cmd(command, options)` inside `CommandHelper.get_command(exec_path, params)`, so if we need to call it from other place we don't have to write it in two lines. What you think ?